assets/scripts/dialogs/Analytics/RadarGraph.jsx

```diff
@@ -0,0 +1,104 @@
+// From example docs
+import React, { Component } from 'react'
+import PropTypes from 'prop-types'
+import { RadarChart, CircularGridLines } from 'react-vis'
+
+const DOMAIN_SMALL = [
+  { name: 'commercial', domain: [0, 100], tickFormat: t => t },
+  { name: 'pedestrian', domain: [0, 100] },
+  { name: 'commuting', domain: [0, 100] },
+  { name: 'congestion', domain: [0, 100] }
+]
+const DOMAIN_LARGE = [
+  { name: 'commercial', domain: [0, 500], tickFormat: t => t },
+  { name: 'pedestrian', domain: [0, 500] },
+  { name: 'commuting', domain: [0, 500] },
+  { name: 'congestion', domain: [0, 500] }
+]
+
+const combineValues = (arr, keys) => {
+  const arrCopy = Array.from(arr)
+  const keyFunc = (total, item) => {
+    keys.forEach(k => {
+      total[k] = (total[k] + item[k])
+    })
+    return total
+  }
+
+  return arrCopy.reduce(keyFunc)
+}
+
+// Animated Radar widget which shows data for street analytics
+export default class AnimatedRadar extends Component {
+  static propTypes = {
+    data: PropTypes.array,
+    dataKeys: PropTypes.array
+  }
+
+  constructor (props) {
+    super(props)
+    this.toggleCombined = this.toggleCombined.bind(this)
+  }
+
+  state = {
+    showCombined: false
+  }
+
+  toggleCombined () {
```

Comment:
We can write this like so:
```
toggleCombined = () => {
```
This avoids having to use the `.bind(this)` syntax on line 40.